test/buildenv.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
-
+#
+# Copyright (C) 2018 Ultimaker B.V.
+# Copyright (C) 2018 Raymond Siudak <raysiudak@gmail.com>
+# Copyright (C) 2018 Olliver Schinagl <oliver@schinagl.nl>
+#
+# SPDX-License-Identifier: AGPL-3.0+
 set -eu
```

Comment:
:+1: on the license addition, but missing whitespace

(I'm thinking now that you have done this on EMP-302 or added the changes to this PR as well. Both are fine, but then you'll have to pull in the rework from the EMP-302 on these things too, if you want to merge those with this branch)

Reply:
Yeah part of EMP-302